design-proposals/dynamic-primary-pod-nic-name.md

```diff
@@ -0,0 +1,481 @@
+Support Dynamic Primary Pod NIC Name
+=
+
+# Summary
+
+This proposal proposes a mechanism to dynamically determine the primary pod interface for KubeVirt virtual machines (VMs).
+The goal is to move away from the hard coded `eth0` interface and allow flexible, environment-driven interface selection.
```

Comment:
Maybe this is useful for your design doc: the only requirement I could find for the pod networking (related to interface names) is the loopback interface: https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/#loopback-cni